source/help/messaging/formatting-text.rst

```diff
@@ -235,3 +235,19 @@ Create a table by placing a dashed line under the header row and separating the
 Renders as:
 
 .. image:: ../../images/markdownTable1.PNG
+
+
+Math Formulas
```

Comment:
Thanks for the explaination, that makes sense, I'm okay with keeping it separate.